pages/linux/ffuf.md

```diff
@@ -0,0 +1,24 @@
+# ffuf
+
+> Subdomain and directory discovery tool.
+> More information: <https://github.com/ffuf/ffuf>.
+
+- Directory discovery using a wordlist on a target website colorized and verbose:
```

Comment:
All descriptions should be written in [`imperative`](https://en.wikipedia.org/wiki/Imperative_mood#English). For example:
```suggestion
- Find subdomains of a target website using a wordlist and show colorized and verbose output:
```

Reply:
I understand what you are getting at.  
However, the line 6 & 8 description/command pair is a directory discovery execution and not a subdomain discovery execution.
I've added in an extra command for subdomain discovery, so the current program description is more accurate.  This will take the number of examples to 6 from the recommend 5.

If the "discovery" word choice is poor, I am open to suggestions.
My current rationale is:
"Find" is more akin to locating something that you know exists and do not know where its current whereabouts are.
Whereas "discovery" is identifying or interacting with something that there was no previous knowledge of.

In my experience, the tool/program is used as a reconnaissance tool, on unfamiliar systems, hence the "discovery" stance.

addressed in 60d5192b7124796d1ce615d85a0cb1de0d18135e

Reply:
Ah, okay. But my point was that the sentences should be written like `Discover directories using [...]`, instead of `Directory discovery [...]`, since that's the syntax we use in all pages. I can't really do that accurately enough since I don't understand the command well enough.

Reply:
Gotcha.
Edited the command descriptions to be in line with your comments.

addressed in 7564c8f458e27155ea9e882cf70b400f89895338